modules/programs/thunderbird.nix

```diff
@@ -124,6 +124,28 @@ let
     '') prefs)}
     ${extraPrefs}
   '';
+
+  filterIniHeader = ''
+    version="9"
+    logging="no"
+  '';
+
+  mkFilterToIniStr = f: ''
+    name="${f.name}"
```

Comment:
I guess, if you are explicitly listing these, maybe you should also document them separately, alternatively I am moderately sure nixpkgs has some generators for things like this, maybe ini?